src/quacc/recipes/espresso/dos.py

```diff
@@ -125,6 +129,49 @@ def projwfc_job(
     )
 
 
+@job
+def bands_pp_job(
```

Comment:
Is there any reason to have a separate post-processing job? Why not put the post-processing step within `band_job`? I assume that almost all users will want to do post-processing.

My recommendation would be to do both steps in `band_job`, removing the need for this separate job.